src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
+    // Intentional errors added below
```

Comment:
5. Error 5: Inconsistent naming convention
   - The `Subtract` method name starts with an uppercase alphabet, which is inconsistent with the standard Java naming conventions. Method names should start with a lowercase letter.